io.github.smolblackcat.Progress.json

```diff
@@ -0,0 +1,152 @@
+{
+    "id": "io.github.smolblackcat.Progress",
+    "runtime": "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "46",
+    "command": "progress-tracker",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "modules": [
+        {
+            "name": "gtkmm-4-10",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dmaintainer-mode=false"
+            ],
+            "cleanup": [
+                "*.a",
+                "*.la",
+                "/include",
+                "/lib/gdkmm-4.0",
+                "/lib/gtkmm-4.0",
+                "/lib/pkgconfig",
+                "/man",
+                "/share/aclocal",
+                "/share/man",
+                "/share/pkgconfig"
+            ],
```

Comment:
You should add the common ones to a top level cleanup, to reduce duplication:

```json
"*.a",
"*.la",
"/include",
"/lib/pkgconfig",
"/man",
"/share/aclocal",
"/share/man",
"/share/pkgconfig"
```

Then remove individual ones from the modules below.